src/layout/Layout/Layout.tsx

```diff
@@ -1,13 +1,16 @@
 import { Outlet } from 'react-router-dom';
 import { Header } from '../Header/Header';
 import { Footer } from '../Footer/Footer';
+import { Container } from '@mui/material';
 
 export const Layout = () => {
     return (
         <>
             <Header />
-            <main className="container">
-                <Outlet />
+            <main style={{ flexGrow: '1' }}>
```

Comment:
Can we separate styles to another file?

Reply:
Yes. I have updated it in the recent commit.